src/matchers/toBeAfterOrEqualTo/predicate.test.js

```diff
@@ -1,18 +0,0 @@
-import predicate from './predicate';
-
-const EARLIER = new Date('01/09/2019');
```

Comment:
_toBeAfterOrEqualTo/index.test.js_ has all the dates formatted in ISO 8601. So deleting this file deletes the only tests with this format. Not sure if we care about that.

Reply:
I don't think we care what format the date is constructed from just purely the difference between two dates meets the matcher (`toBeAfter`/`toBeBefore` etc)